thorlcr/thorutil/thmem.cpp

```diff
@@ -1237,6 +1237,8 @@ CThorSpillableRowArray::CThorSpillableRowArray(CActivityBase &activity)
     : CThorExpandingRowArray(activity)
 {
     initCommon();
+    commitDelta = CommitStep;
+    throwOnOom = false;
```

Comment:
In other constructor, commitDelta is set as a constructor initializer.
And throwOnOom was set in base class, but should have been overridden to match intended value (set via base constructor call in other CThorSpillableRowArray constructor).